src/main/java/com/kodedu/config/EditorConfigBean.java

```diff
@@ -778,7 +778,7 @@ public JsonObject getJSON() {
                 .add("previewScreenY", getPreviewScreenY())
                 .add("previewScreenWidth", getPreviewScreenWidth())
                 .add("previewScreenHeight", getPreviewScreenHeight())
-                .add("foldStyle", getFoldStyle().name())
+                .add("sStyle", getFoldStyle().name())
```

Comment:
@gillesB  Was this an intended change?

Reply:
Sorry, this must have been an unintentional change. I see no reason, why I changed it. Anyway I will revert it in the next commit.